client.go

```diff
@@ -141,8 +141,8 @@ func (c *Client) CheckAPIKey() ([]byte, error) {
 // See: https://docs.cohere.ai/generate-reference
 // Returns a GenerateResponse object.
 func (c *Client) Generate(opts GenerateOptions) (*GenerateResponse, error) {
-	if opts.NumGenerations == 0 {
-		opts.NumGenerations = 1
+	if opts.NumGenerations == nil || *opts.NumGenerations == 0 {
+		opts.NumGenerations = Int(1)
```

Comment:
Can we let the backend handle this instead?

Reply:
yeah sure, it'll default to 1 on the backend.

Reply:
sweet, so let's send nil instead then